docs/en/contributing/working.md

```diff
@@ -20,39 +20,7 @@ PR comments should include the issue number per the [github closing issue keywor
 
 One or more collaborators are automatically assigned to review each PR based on areas of [ownership](https://github.com/Microsoft/fast-dna/blob/master/.github/CODEOWNERS).
 
-**`<type>`** is required to be at least one of the following:
-
-- chore: A change that does not impact distributed packages
-- feat: Adding, updating, or removing a feature
-- fix: A bug fix.
-
-**`<description>`** is required and speaks to what the user gets from this PR:
-
-- Be concise
-- Use lowercase
-- Use imperative, present tense (e.g. `add` not `adds`)
-- Do not end with a period. Period.
-- Avoid redundant words
-
-Accepted title format
-
-```comment
-<type>: <description>
-```
-
-Example titles
-
-```comment
-feat: add a border radius to MSFT button
-chore: add unit-tests for all components
-fix: update MSFT design system to use 3px border radius
-```
-
-### Pull request template
-
-In addition to providing a properly formatted title for your pull request, we offer a template that will help guide you toward appropriate best practices and processes as well as provide additional information which will give those reviewing your request the context they need.
-
-Open the [pull request template](https://github.com/Microsoft/fast-dna/blob/master/.github/pull_request_template.md).
+At the time a PR is created, we use a [template](https://github.com/Microsoft/fast-dna/blob/master/.github/pull_request_template.md) that will help guide you toward appropriate best practices and processes as well as provide additional information which will give those reviewing your request the context they need.
```

Comment:
Try something like: 
"When a peer review is created, a [template](https://github.com/Microsoft/fast-dna/blob/master/.github/pull_request_template.md) is provided that includes the best practices and processes for a review message.